docs/cis_self_assessment.md

```diff
@@ -697,22 +697,18 @@ The AlwaysAdmit admission controller was deprecated in Kubernetes v1.13. Its beh
 
 **Result:** Pass
 
-**Remediation:**
-Edit the API server pod specification file `/etc/kubernetes/manifests/kube-apiserver.yaml`
-on the master node and either remove the `--enable-admission-plugins` parameter, or set it to a
-value that does not include `AlwaysAdmit`.
-
 **Audit:**
+Run the below command on the master node.
 
-```
+```bash
 /bin/ps -ef | grep kube-apiserver | grep -v grep
 ```
 
-**Expected result**:
+Verify that if the `--enable-admission-plugins` argument is set, its value does not include `AlwaysAdmit`.
 
-```
-'NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota,NodeRestriction,Priority,TaintNodesByCondition,PersistentVolumeClaimResize,PodSecurityPolicy,EventRateLimit' not have 'AlwaysAdmit' OR '--enable-admission-plugins' is not present
-```
+**Remediation:**
+By default, RKE2 only sets `NodeRestriction,PodSecurityPolicy` as the parameter to the `--enable-admission-plugins` argument.
+To configure this, follow the Kubernetes documentation and set the desired limits in a configuration file. Then refer to RKE2's documentation to see how to supply additional api server configuration via the kube-apiserver-arg parameter.
```

Comment:
This control seeks to ensure AlwaysAdmit is NOT set. I don't think we need to give them this line about how to conigure it. This one can have the "No manual rem..." bit